### ISSUE TITLE
No CSV

### ISSUE THREAD

**Issue by vpatel27** [penrosestudio/barclays-bank-pdf-to-csv#4]
Hi

I might be being a bit thick here...so please excuse me if I am.
I dragged a pdf into the drop zone, the screen populated with transactions and cover sheet.
But how do I get the actual CSV?

**Comment by AlfieGreen** [comment]
I'm having the same problem... Am I doing something wrong?

**Comment by crobbins215** [comment]
Same, i think I must be doing something wrong

**Comment by jayfresh** [comment]
if you use the in-browser tool it doesn't give you a CSV - you can copy the transactions into a CSV or spreadsheet. If you want the CSV file output, you can use the command-line tool - see [Usage under node.js](https://github.com/penrosestudio/barclays-bank-pdf-to-csv#usage-under-nodejs)

**Comment by bigfatfrog67** [comment]
This is great, shame I didn't have when I did last years accounts ;-)
I can't see how I can copy and paste the resulting data from the web page as it isn't in csv format, just plain text.
Also, I don't understand how to use it from the cli?  What does "under node.js" mean?
Thanks

**Comment by jayfresh** [comment]
Hi all, if you are using the webpage to create the transactions list, it is created in CSV format, so you can copy and paste the transactions into a text file and save it with a .csv extension - that will be your csv file.

If you are wanting the tool to generate a CSV, see the readme for the instructions. It is a bit trickier to use as you need to have node.js installed and run it from the command line.

**Comment by bigfatfrog67** [comment]
There are NO commas in the webpage version, if i copy and paste that i get a SPACE separated file!

**Comment by jayfresh** [comment]
are they not tabs?

**Comment by bigfatfrog67** [comment]
No, just a box below where you drop pdf which shows the output but in free text

**Comment by bigfatfrog67** [comment]
Using Firefox on Windows

**Comment by mrghouston** [comment]
Im not really clued up on all the jargon...I managed to get the pdf converted nicely but im really struggling to convert it to a csv.. Im using a mac..can you go over again slowly for me ?

**Comment by jayfresh** [comment]
Sorry! OK, I'll try again...

If you use the in-browser convertor, the transactions that appear actually have *tabs* separating the date from the description and the transaction values. If you copy this text into a text file and upload that to Google Spreadsheets, or copy it into Excel, you'll be able to save/export that as a CSV file.

Make sense? Let me know if not!

**Comment by mrghouston** [comment]
Jonathan ,

Many thanks for taking the time to respond to me. I have the nightmare
scenario of manually inserting 18 months transactions into my accounting
software because Barclays will not give me the right format..as you know
!..Unbelievable considering they also sold me freeagent accounting software
at the time of opening the account.
I was delighted to see that you had provided a solution although I have now
spent days trying to get it working.. I think the problem is when I create
the text file. , because when I then import it to Google sheets it all goes
in the first column without separation..When I then export the csv it will
not upload and states the data needs to be in separated columns. Is there
something simple Im missing ?..Im using text edit on the Mac to copy the
data to txt format.

Gerry

**Comment by jayfresh** [comment]
Aha, what you need to do is select "Tab" as the delimiter when you import the file into Google Spreadsheets. I've just double-checked this and it correctly puts the data into multiple columns. Good luck!

**Comment by mrghouston** [comment]
Worked a treat !!....so grateful to you...saved me sooo much work...many
thanks !

**Comment by crobbins215** [comment]
I copy and paste everything into barclays.txt and when I import it into google sheets it changes to a text document, when I import it into excel it imports every space into a separate column, are you able to do step by step?

Also only 3 out of the 5 pdf's worked
